libs/template/template.go

```diff
@@ -38,7 +38,7 @@ const (
 var databricksTemplates = []Template{
 	{
 		name:        DefaultPython,
-		description: "The default Python template for Notebooks / Delta Live Tables / Workflows",
+		description: "The default Python template for Notebooks / Lakeflow Declarative Pipelines / Workflows",
```

Comment:
This should say "Notebooks and Lakeflow"; Workflows was subsumed by Lakeflow

Reply:
https://github.com/databricks/cli/pull/3487/files